_security-analytics/sec-analytics-config/detectors-config.md

```diff
@@ -13,64 +13,72 @@ Security Analytics provides the options and functionality to monitor and respond
 
 You can define a new detector by naming the detector, selecting a data source and detector type, and specifying a detector schedule. After defining a detector, you can also configure field mappings and set up alerts. Follow the steps in this section to accomplish all three of these setup tasks.
 
-1. On the Detectors page, select the **Create detector** button. The Define detector page opens.
-1. Give the detector a name and, as an option, add a description for the detector. 
-1. In the Data source section, select the dropdown arrow and select a source for the log data.
-1. In the threat detection type section, select the data type. The Sigma security rules associated with the log data are automatically populated in the Detection rules section below it.
-<img src="{{site.url}}{{site.baseurl}}/images/Security/detector_rules.png" alt="Selecting threat detector type to auto-populate rules" width="80%">
+1. On the **Threat detectors** page, choose **Create detector**. The **Define detector** page opens.
+1. In **Detector details**, give the detector a name and, as an option, add a description for the detector. 
+1. In the **Data source** section, select the dropdown arrow and choose one or multiple sources for the log data.
 
-    You can skip the next step for mapping rules if you are satisfied with those automatically populated by the system. Otherwise, go to the next step to specify select rules.
+    When multiple data sources are selected, the logs must be of the same type. We recommend creating separate detectors for different log types.
```

Comment:
I think this might be clearer if you replace "When" with "If" because it isn't an inevitability:

```
If multiple data sources are selected, the logs must be of the same type.
```